lldb/source/Plugins/ABI/RISCV/ABISysV_riscv.cpp

```diff
@@ -643,9 +644,9 @@ bool ABISysV_riscv::CreateFunctionEntryUnwindPlan(UnwindPlan &unwind_plan) {
   unwind_plan.Clear();
   unwind_plan.SetRegisterKind(eRegisterKindDWARF);
```

Comment:
Since we specify here that we are using DWARF registers, then the change below is correct,